src/router/index.ts

```diff
@@ -8,26 +8,24 @@ import Page404 from '@/containers/404.vue';
 
 Vue.use(Router);
 
-const baseURL = process.env.BASE_URL;
-
 const router = new Router({
   mode: 'history',
   routes: [
     {
-      path: `${baseURL}/login`,
+      path: '/login',
       name: 'Login',
       component: Login,
     },
     {
-      path: `${baseURL}/`,
+      path: '/',
       name: 'Dashboard',
       component: Dashboard,
       meta: {
         requiresAuth: true,
       },
     },
     {
-      path: `${baseURL}/*`,
+      path: '/*',
       name: '404',
       component: Page404,
     },
```

Comment:
the base_url is used as a hack for github pages deployment. Now it's no longer needed.

Reply:
What was the downside of using `baseURL` (i.e. env variables)?

Reply:
No downside. It's simply unnecessary now.